src/main/java/seedu/address/ui/Commands.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.ui;
+
+/**
+ * Enum representing the various commands available in the application.
+ * Each command has a name, an example format, and associated prefixes for parameters.
+ */
+public enum Commands {
+    ADD("add", "add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]",
+            new String[]{"n/", "p/", "e/", "a/", "t/"}),
+    DELETE("delete", "delete INDEX", new String[]{}),
```

Comment:
Since right now we are trying to allow for 2 types of delete commands (delete INDEX / delete NAME), is it possible to allow for a more flexible command format?